p4-16/spec/P4-16-spec.mdk

```diff
@@ -2852,6 +2870,10 @@ operations simply "wrap around", similar to C operations on unsigned values.
 Hence, attempting to represent large values using `W` bits will only keep
 the least-significant `W` bits of the value.
 
+P4 also supports saturating arithmetic (addition and subtraction) for signed
```

Comment:
For signed integers I suspect we want this to work even when adding negative values.

Reply:
There is nothing in the text that suggests it should not. The minimum value is `-2^(W-1)`. Do you think we need to add additional clarification?